server/routes/suggestions/suggestionsController.ts

```diff
@@ -9,12 +9,8 @@ export default class SuggestionsController {
   }
 
   applySuggestions(req: Request, res: Response) {
-    if (req.query.firstName) {
-      req.session.prisonerSearchForm.firstName = req.query.firstName as string
-    }
-    if (req.query.lastName) {
-      req.session.prisonerSearchForm.lastName = req.query.lastName as string
-    }
+    req.session.prisonerSearchForm.firstName = req.query.firstName as string
```

Comment:
need to set first and last name in case one is empty and we are swapping first and last name

Reply:
what happens if they accept an age suggestion though and there is a first name - won't it be cleared out?

Reply:
you are totally correct - same for first and last name - will ensure tests cover this...just adding more controller tests too